src/sentry/flags/providers.py

```diff
@@ -50,28 +50,30 @@ class InvalidProvider(Exception):
     ...
 
 
-class LaunchDarklyItemSerializer(serializers.Serializer):
-    """Docs reference: https://apidocs.launchdarkly.com/tag/Audit-log/#operation/getAuditLogEntry"""
+def handle_provider_event(
+    provider: str,
+    request_data: dict[str, Any],
+    organization_id: int,
+) -> list[FlagAuditLogRow]:
+    match provider:
+        case "launchdarkly":
+            return handle_launchdarkly_event(request_data, organization_id)
+        case _:
+            raise InvalidProvider(provider)
+
 
+"""LaunchDarkly provider."""
+
+
+class LaunchDarklyItemSerializer(serializers.Serializer):
     accesses = serializers.ListField(required=True)
     date = serializers.IntegerField(required=True)
     member = serializers.DictField(required=True)
     name = serializers.CharField(max_length=100, required=True)
     description = serializers.CharField(required=True)
 
 
-"""
-LaunchDarkly has a lot more flag actions than what's in our
-ACTION_MAP. The "updated" action is the catch-all for actions
-that don't fit in the other buckets.
-
-We started out with a few actions that we think would be useful
-to accept. All other actions will not be logged
-to the audit log. This set of actions is subject to change.
-"""
-
-# A subset chosen from https://docs.launchdarkly.com/home/account/role-actions#feature-flag-actions
```

Comment:
Can we keep this link? Personally I like having this in the code to quickly refer to, in case this changes or we want to learn more about a specific action

Reply:
Ok, just saw your comment on my other PR, let's remember to link this in our docs instead